### ISSUE TITLE
Unexpected aztec code size limitation

### ISSUE THREAD

**Issue by muued** [adamchainz/treepoem#476]
### Python Version

3.10.0

### Package Version

3.18.0

### Description

Aztec codes seem to have a hard capacity limitation of 1248 bytes. Online resources suggest the maximum capacity to encode bytes is at 1914 bytes (e. g., [Wikipedia](https://en.wikipedia.org/wiki/Aztec_Code#Structure)). The capacity of 1914 bytes also includes the error correction words. Using the default error correction settings, we should end up with a capacity constraint of around 1550 bytes (1914 / 1.23). Encoding 1248 bytes should thus be possible.

The following fails with `treepoem.TreepoemError: GPL Ghostscript 9.55.0: Unrecoverable error, exit code 1`
```
import treepoem
treepoem.generate_barcode('azteccode', bytes.fromhex('00' * 1248))
```

However, using one less byte, I can still dial up the eclevel to 49% before the capacity is exhausted:
```
import treepoem
treepoem.generate_barcode('azteccode', bytes.fromhex('00' * 1247), {'eclevel': 49})
```

So the limitation above seems to make no sense. Or am I missing something?

**Comment by adamchainz** [comment]
I'm afraid that I have no idea about any specific barcode formats. This library only wraps BWIPP. This sounds like a problem to search for and maybe report upstream in the BWIPP project: https://github.com/bwipp/postscriptbarcode .

**Comment by muued** [comment]
The problem was indeed upstream and is fixed in [Release 2023-07-12](https://github.com/bwipp/postscriptbarcode/releases/tag/2023-07-12)

**Comment by adamchainz** [comment]
Upgraded in #494, released in 3.21.0.
